backend/src/main/java/com/bakdata/conquery/resources/api/ConceptsProcessor.java

```diff
@@ -196,17 +180,14 @@ public ResolvedConceptsResult resolveFilterValues(SelectFilter<?> filter, List<S
 			}
 		}
 
-		return new ResolvedConceptsResult(null, new ResolvedFilterResult(filter.getConnector().getId(), filter.getId(), out), openSearchTerms);
-	}
+		final ConnectorId connectorId = filter instanceof Filter asFilter ? asFilter.getConnector().getId() : null;
+		final Id<?> id = ((Identifiable<?>) filter).getId();
+
```

Comment:
`filter` bitte noch in `searchable` umbenennen, sonst sieht es komisch aus. Es scheint gerade so, dass alle Implementierungen von Searchable schon Identifiable sind und auch einem Connector untergeordnet sind.